sdl2-vga-terminal/src/tui/utils.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <string>
+
+namespace tui
+{
+	/**
+	 * @width  it is the size of the line.
+	 * @return the position where it start and end the middle string
+	 */
+	std::pair<uint8_t, uint8_t> align3Strings(const std::string& left, const std::string& middle, const std::string right, const uint8_t width);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'right' should be passed by const reference.](https://app.codacy.com/manual/Raffaello/sdl2-vga-terminal/pullRequest?prid=5472420)